src/Libraries/Microsoft.Extensions.AI.Evaluation.Quality/SingleNumericMetricEvaluator.cs

```diff
@@ -49,8 +37,16 @@ Your response should be a single character containing only the evaluation score.
         in your response besides the evaluation score.
         """;
 
-    // TASK: Explore using structured output and providing a JSON schema to better enforce the LLM response format
-    // requirements above. Tracked by https://github.com/dotnet/extensions/issues/5888.
+    private readonly ChatOptions _chatOptions =
+        new ChatOptions
+        {
+            MaxOutputTokens = 1,
+            Temperature = 0.0f,
+            TopP = 1.0f,
+            PresencePenalty = 0.0f,
+            FrequencyPenalty = 0.0f,
```

Comment:
Have you found that setting these is helpful? Are these not the default values used in most services?

Reply:
That's a good question. `MaxOutputTokens` is certainly useful in this context. The rest were copied over from the original python-based implementation. I am not sure what the defaults for these parameters are, and whether the defaults are the same across different models / providers. Most of our real-world usage / testing has been against Azure Open AI, and perhaps some of these are indeed redundant. However, given that the deployed evaluations have been working well, I'd rather not change it at this point unless there is a strong reason to...